### ISSUE TITLE
[BUG] Version 6.5.4 blocks Desktop Environment

### ISSUE THREAD

**Issue by pmanu93** [getinsomnia/insomnia#1546]
- Insomnia Version: 6.5.4
- Operating System: Linux Mint 17.3 64bit (Cinnamon 2.8.8)
## Details

After trying to execute insomnia, the desktop environment (cinnamon) blocks. Only mouse moves.

**Comment by pmanu93** [comment]
No problem with version 6.5.3 tho.

**Comment by gschier** [comment]
Hmm, the only thing that really changed was 6.5.4 included new icon sizes. Can you try starting it from the command line and seeing if that shows any errors?

**Comment by pmanu93** [comment]
Starting it via command line, does not work. Even in 6.5.3..

This is the output from the command line:
![Captura de ecra de 2019-06-04 10:34:45](https://user-images.githubusercontent.com/15691766/58868511-677d2200-86b4-11e9-9d95-416bdd3ad14e.png)

It actually opens a window, but with artifacts: 
![ezgif-4-7fbcc43d4e56](https://user-images.githubusercontent.com/15691766/58868697-c347ab00-86b4-11e9-810f-e5ffd7a3cb04.gif)

**Comment by pmanu93** [comment]
Update:
If I wait a little without closing the window, a message saying "Insomnia has become unresponsive" (or something similar), and right after that, my system halts (like when I open via start menu)

The only difference between command line and start menu shortcut, is the fact that in command line opens a window and only after a few seconds the system halts. Via start menu system halts before showing any window.

**Comment by shutterstein** [comment]
We are experiencing this as well and usually end up having do to a hard boot.  It seems like this started for me a couple of versions ago.  It doesn't happen every time but seems more likely if I have Spring Tool Suite up and running or if I launch Insomnia and try to do anything else while insomnia is trying to start up.  It freezes everything but the mouse up and it all becomes unresponsive.  I've resorted at this point to starting Insomnia first thing after a boot and then starting other applications after it is fully up.

Operating System: Linux Mint 18.2 Cinnamon 64-bit
Cinnamon Version: 3.4.6
Linux Kernel: 4.8.0-53-generic

Current info in Insomnia:
Version 6.5.4
Shell 3.0.4
Node 10.2.0
V8 6.6.346.32
Architecture x64

libcurl/7.61.1 OpenSSL/1.1.0h zlib/1.2.11 nghttp2/1.29.0

**Comment by shutterstein** [comment]
So I've looked more into what is going on.  I downloaded and have attempted to run 6.5.4, 6.5.3 and 6.5.1.  

Both versions above 6.5.1 fail on me when the application is starting up about 95% of the time (even on a fresh reboot with nothing running).  

At first I had installed it using snap so I uninstalled it thinking it might be that I had installed it through there.  That seems to have had no relevance to the issue.  It hangs on me every time after this step in the console when starting the application via command-line:
`[localstorage] Initialized at /home/benst/.config/Insomnia/localStorage`

See the attached image I took for 2 attempts on 6.5.1 that were successful and then a failed attempt that froze my system on 6.5.3.

![20190625_153815](https://user-images.githubusercontent.com/4976502/60136350-2df28080-9761-11e9-854e-4942382fac6c.jpg)

6.5.1 seems to work every time just fine form me so I'm sticking with that for now so I don't have to keep hard-booting my machine.

**Comment by berni2288** [comment]
Insomnia Version: 6.6.2
Linux Mint 19.2

The problem still exists, please reopen this issue!

**Comment by pmanu93** [comment]
Yap. Updated to 7.0.1 and problem still happens...
I can only use Insomnia when deleting "~/.config/Insomnia" folder every time I want to use..

**Comment by sophiekuehn** [comment]
I have the same issue and reverted back to 6.5.1 now. Thanks to shutterstein for that workaround.

**Comment by gschier** [comment]
Since this is old, can someone create a new issue?

**Comment by tvelich** [comment]
https://github.com/Kong/insomnia/issues/1922
